script.bluetooth.delay/addon.py

```diff
@@ -99,24 +79,32 @@ def speakers():
 def main():
 
     if arg == "reset":
-        xbmc.executebuiltin('PlayerControl(stop)')
-        Addon.setSettingBool('state', 1)
-        reset1()
+        if (xbmc.getCondVisibility('Player.HasVideo') == False):
+            xbmc.executebuiltin("PlayMedia("+path+"reset.mp4,1)")
+        xbmc.executebuiltin('ActivateWindow(busydialognocancel)')
+        time.sleep(2)
+        for x in range (800):
```

Comment:
To change the delay value of a video - you must play a video.
Otherwise you do not have access to osdaudiosettings.
The idea is to reset the delay whenever a new value is set.

Reply:
This really sounds like core limitations that should be fixed/attempted there

Reply:
But yeap, shipping videos (or any binary file) is not allowed sorry